hidimstat/estimator/_utils/u_Dnn_learner.py

```diff
@@ -0,0 +1,657 @@
+
+import numpy as np
+import copy
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchmetrics import Accuracy
+from sklearn.metrics import log_loss, mean_squared_error
+from sklearn.preprocessing import StandardScaler
+
+
+
+def create_X_y(
```

Comment:
This should probably go to a `_utils` module ?

Reply:
This will be addressed by the issue #81.